src/main/scala/com/linkedin/feathr/offline/config/location/GenericLocation.scala

```diff
@@ -98,30 +98,18 @@ object GenericLocationAdHocPatches {
     location.conf.foreach(e => {
       ss.conf.set(e._1, e._2)
     })
-
```

Comment:
You need to add this section back in your PR, your commit is conflicting with the main branch.